README.md

```diff
@@ -53,38 +48,147 @@ See the [Docker documentation](./docs/Docker.md) to get up and running
+  ```
+
+  To run the smoke tests against a deployed server, make sure you set up a `.env` file with the right configuration values, see [monitor_config.rb](spec/support/monitor/monitor_config.rb) for the full list of environment variables used. The script below will `source` that file and add the variables to the environment.
+
+  ```bash
+  MONITOR_ENV=INT ./bin/smoke_test --remote
+  ```
+
+#### Speeding up local development and testing
+
+  To automatically run the test that corresponds to the file you are editing,
+  run `bundle exec guard` with the env var `GUARD_RSPEC_CMD` set to your preferred
+  command for running `rspec`. For example, if you use [Zeus](https://github.com/burke/zeus),
+  you would set the env var to `zeus rspec`:
+  ```console
+  GUARD_RSPEC_CMD="zeus rspec" bundle exec guard
+  ```
+
+  If you don't specify the `GUARD_RSPEC_CMD` env var, it will default to
+  `bundle exec rspec`.
+
+  We also recommend setting up a shell alias for running this command, such as:
+  ```console
+  alias idpguard='GUARD_RSPEC_CMD="zeus rspec" bundle exec guard'
+  ```
+
+#### Viewing email messages
+
+  To view email messages, Mailcatcher must be running. You can check if it's
+  running by visiting http://localhost:1080/. To run Mailcatcher:
+
+  ```
+  $ mailcatcher
+  ```
+
+  If you would like to run the application on a different port:
+
+  * Change the port number for `mailer_domain_name` and `domain_name` in `config/application.yml`
+  * Run the app on your desired port like `make run PORT=1234`
+
+#### Translations
+
+  Login.gov translates the IdP into English, French and Spanish. To help us handle extra newlines and make sure we wrap lines consistently, we have a script called `./scripts/normalize-yaml` that helps format YAML consistently. After importing translations (or making changes to the *.yml files with strings, run this for the IdP app:
+
+  ```
+  $ make normalize_yaml
+  ```
+
+  [mac-test-passphrase-prompt]: mac-test-passphrase-prompt.png "Mac Test Passphrase Prompt"
+
+  If you would like to preview the translations on a particular page, add
+  `?locale=es` to the end of the URL, such as `http://localhost:3000/?locale=es`.
+  Currently, you'll need to add `?locale=es` to each URL manually.
+
+#### Viewing outbound SMS messages and phone calls
+
+  To see outbound SMS messages and phone calls, visit `http://localhost:3000/test/telephony`.
+
+#### Adding a custom proofing vendor
+
+    We've added the ability for you to use a custom proofing vendor when developing locally. Here is an example of how you would configure that in your local development environment:
+
+    Check out the private repository for `somevendorname`
+
+    ```
+    $ cd vendor
+    $ git clone git@github.com:18F/identity-somevendorname-api-client-gem.git somevendorname
+    ```
+
+    Add appropriate vendor environment variables to `config/application.yml` -- see a member of the
+    login.gov team for credentials and other values.
```

Comment:
This is not a common workflow anymore, I'd consider just removing it?

Reply:
Talked to the team and 💯 to removing. Good call!

Reply:
Updated here 226713e